recipe/meta.yaml

```
@@ -1,35 +1,36 @@
 {% set name = "chainladder" %}
-{% set version = "0.8.14" %}
-
+{% set version = "0.8.18" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/chainladder-{{ version }}.tar.gz
-  sha256: 346c8a71d5d7188b22203da9a49ff35f93e00129bf17b2ea2f45ccc7bd5588ac
+  sha256: 56eb8bd44d46ca48680122f85d96dba6f33d7a82393a6468786b061f266161dc
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  # skip py39 because of missing sparse
+  skip: True  # [py<39 or s390x]
 
 requirements:
   host:
     - pip
-    - python >=3.5
+    - python
+    - setuptools
+    - wheel
   run:
     - joblib
-    - numpy >=1.12.0
-    - pandas >=0.23.0
-    - python >=3.5
-    - scikit-learn >=0.23.0
+    - python
+    - numpy >=1.12
+    - pandas >=0.23
+    - scikit-learn >=0.23
     - scipy
     - matplotlib
```

```suggestion
    - matplotlib-base
```
I think the additional `pyqt` isn't needed